### PR TITLE
9.0 base technical features

### PR DESCRIPTION
**Pull request by StefanRijnhart**
Make technical features persistent by adding users to a new group.

### CODE REVIEW

File: base_technical_features/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016 Opener B.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Technical features group",
+    "summary": "Access to technical features without activating debug mode",
+    "version": "8.0.1.0.0",
```
Comment (RoelAdriaans) [inline base_technical_features/__openerp__.py]: Shoudn't this be 9.0?

Comment (StefanRijnhart) [inline base_technical_features/__openerp__.py]: Fixed, thanks!

**Comment by parthivgls** [comment]
superb :+1:

**Comment by ghost** [comment]
+1

**Comment by elicoidal** [comment]
magnifique !
:+1:

**Comment by StefanRijnhart** [comment]
What do you think, do we want this as a user preference (for users that are already in base.group_no_one of course)?

**Comment by max3903** [comment]
:+1: for this pr and to have this as a user preference

**Comment by StefanRijnhart** [comment]
Yes, I would prefer that as well.

**Comment by StefanRijnhart** [comment]
It's now a user preference.

**Comment by nhomar** [comment]
:+1:

**Comment by moylop260** [comment]
Could you add `images` key in manifest file?

**Comment by StefanRijnhart** [comment]
@moylop260 I was wondering about that message from the OCA pylint plugin. What is this 'images' key for? I could not find any references in the Odoo documentation or even code but maybe I missed something.

**Comment by StefanRijnhart** [comment]
@moylop260 I think I found it! It's used in the Apps store to indicate the image that should be used in the kanban representation of the module. I don't think my ugly screenshot qualifies for that (also see l10n_es on the Apps site!). If I did not miss anything, maybe this key should not be required after all?

**Comment by rafaelbn** [comment]
Thanks! and happy new year! :+1:

**Comment by sebalix** [comment]
Nice!
Should we add the PO template or it's useless?

**Comment by StefanRijnhart** [comment]
That will come from Transifex.

**Comment by sebalix** [comment]
Okay. Tested, :+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**